docs/organizations/changing-your-plan-and-billing.md

```diff
@@ -7,27 +7,29 @@ Each organization on Codacy has a dedicated plan and associated billing. To make
     **If you're using GitHub Marketplace,** make changes to your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
 <!--github-marketplace-end-->
 
-![Plan and billing for a Codacy organization](images/organization-plan-billing.png)
+![Plan and billing for a Codacy organization](images/organization-plan-billing.png) <!--TODO: Update image-->
 
 -   **To upgrade** to the Pro plan click **Choose plan**, choose between monthly or yearly billing, and provide your payment and invoice details
 
 -   **To make changes** to your Pro plan or invoice details click **Edit plan details** or click **Edit invoice details**
 
 If you have any questions or need help with your account, please contact <mailto:support@codacy.com>.
 
-## Accepting new people to your organization
+## Allowing new people to join your organization
 
-**On Codacy Cloud**, organization owners control if team members need an approval before joining their organization. Codacy updates the billing automatically when new users join an organization.
+**On Codacy Cloud**, organization owners control if team members need an approval before joining their organization. Codacy updates your seats automatically when new users join an organization.
 
 !!! note
-    **If you're using GitHub Marketplace,** this configuration isn't available and team members must always wait for an organization owner to manually approve their requests to join the organization.
+    -   **If you're using GitHub Marketplace,** this configuration isn't available and team members must always wait for an organization owner to manually approve their requests to join the organization.
+
+    -   In some **Enterprise plans**, Codacy automatically adds to the organization new people that commit on your private repositories. However, you still need to allow them to join  the Codacy app.
```

Comment:
What do you think adding something like
```suggestion
    -   In some **Enterprise plans**, Codacy automatically adds to the organization new people that commit on your private repositories. However, you still need to allow them to join the Codacy app for them to see and use the UI.
```

Something to try to clarify that they only need to join, if they want to come to the UI

Reply:
Yep, I see what you mean. And this leads me to another question, which may also impact this sentence...

Below, we say that:
"Team members that have already been invited to join the organization are automatically approved"

**Question:** In the Active Pro plan, new people (committers) that had been added to the organization are also automatically approved?

If so, "you still need to allow them to join the Codacy app" it's not really true, right? @machadoit

Reply:
Thanks for the clarification, @machadoit. 👍 
I rephrased the note.

Reply:
Sounds good!